packages/kit/tsup.config.ts

```diff
@@ -8,8 +8,8 @@ export default defineConfig((options: Options) => ({
   entry: ["src"],
   format: ["esm"],
   dts: false,
-  minify: !options.watch,
+  minify: false,
```

Comment:
can we activate minify again?

Reply:
yes we can, I just did it now.

why do you want it minified and not treeshaken?